packages/angular_devkit/build_angular/src/builders/karma/tests/options/aot_spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { execute } from '../../index';
+import { BASE_OPTIONS, KARMA_BUILDER_INFO, describeKarmaBuilder } from '../setup';
+import { BuilderMode } from '../../schema';
+
+describeKarmaBuilder(execute, KARMA_BUILDER_INFO, (harness, setupTarget) => {
```

Comment:
If you want to restrict this to application builder mode only, you might need to add the 3rd argument for this callback (`isApplicationBuilder`) and skip/disable the test if that boolean is false.